tox.ini

```
@@ -43,22 +64,27 @@ deps =
     sunpy: tqdm
     sunpy: zeep
 extras =
-    dev
-commands =
+    tests
+commands_pre =
+    oldestdeps: minimum_dependencies drms --filename requirements-min.txt
+    oldestdeps: pip install -r requirements-min.txt
     pip freeze --all --no-input
-    sunpy: pytest -vvv -s -ra --pyargs sunpy.net.jsoc --timeout=120 --remote-data=any {posargs}
+commands =
+    # To amend the pytest command for different factors you can add a line
+    # which starts with a factor like `online: --remote-data=any \`
+    # If you have no factors which require different commands this is all you need:
+    pytest \
+    -vvv \
+    -r fEs \
+    --pyargs drms \
```

Depending on how we want this to play out we can fold the sunpy job into this command like this maybe?
```suggestion
    !sunpy: --pyargs drms \
    sunpy: --pyargs sunpy.net.jsoc \
```

---

Yeah that would be much better